nautobot_ssot/tests/test_contrib.py

```diff
@@ -228,19 +228,20 @@ class Adapter(NautobotAdapter):
             ip_address = NautobotIPAddress
 
         ip_address_1 = IPAddress.objects.create(
-            host="192.0.2.1", prefix_length=24, status=Status.objects.get(name="Active")
+            host="192.0.2.1", mask_length=24, status=Status.objects.get(name="Active"), parent=self.prefix
         )
 
         self.interface.ip_addresses.set([ip_address_1])
 
         adapter = Adapter()
         adapter.load()
-        diffsync_ip_address = adapter.get(NautobotIPAddress, {"host": "192.0.2.1", "prefix_length": 24})
+        diffsync_ip_address = adapter.get(NautobotIPAddress, {"host": "192.0.2.1", "mask_length": 24})
 
-        self.assertEqual(diffsync_ip_address.assigned_object__app_label, "dcim")
-        self.assertEqual(diffsync_ip_address.assigned_object__model, "interface")
-        self.assertEqual(diffsync_ip_address.assigned_object__device__name, self.device_name)
-        self.assertEqual(diffsync_ip_address.assigned_object__name, self.interface_name)
+        # self.assertEqual(diffsync_ip_address.assigned_object__app_label, "dcim")
+        # self.assertEqual(diffsync_ip_address.assigned_object__model, "interface")
+        # self.assertEqual(diffsync_ip_address.assigned_object__device__name, self.device_name)
+        # self.assertEqual(diffsync_ip_address.assigned_object__name, self.interface_name)
```

Comment:
This still looks like it needs fixing - did something horribly break here causing you to comment it out?

Reply:
Previously IPAddresses were a GFK from ipam.IPAddress to dcim.Interface but now they are ManyToManyField from dcim.Interface to ipam.IPAddress. The `assigned_object` attr is no longer relevant